test/compress_test.js

```diff
@@ -23,6 +23,8 @@ exports.compress = {
       actual.push(entry.path);
     });
     parse.on('close', function() {
+      actual.sort();
+      expected.sort();
```

Comment:
when you give archiver, a set of files it has to queue them to make a sync like api for tasks that happen async (ie stat) so order is not guaranteed